common/lib/xmodule/xmodule/templates/problem/customgrader.yaml

```diff
@@ -24,25 +43,26 @@ data: |
           return test_add(10, ans)
 
       </script>
-
-      <p>Enter two integers which sum to 10: </p>
-      <customresponse cfn="test_add_to_ten">
-              <textline size="40" correct_answer="3" label="Integer #1"/><br/>
-              <textline size="40" correct_answer="7" label="Integer #2"/>
-      </customresponse>
-
-      <p>Enter two integers which sum to 20: </p>
-      <customresponse cfn="test_add" expect="20">
-              <textline size="40" correct_answer="11" label="Integer #1"/><br/>
-              <textline size="40" correct_answer="9" label="Integer #2"/>
-      </customresponse>
-    
-          <solution>
-              <div class="detailed-solution">
-                  <p>Explanation</p>
-                  <p>Any set of integers on the line \(y = 10 - x\) and \(y = 20 - x\) satisfy these constraints.</p>
-                  <p>You can also add images within the solution clause like so:</p>
-                  <img src="/static/images/placeholder-image.png"/>
-              </div>
-          </solution>
+      <fieldset>
+        <legend>Enter two integers that sum to 10.</legend>
+        <customresponse cfn="test_add_to_ten">
+          <textline size="40" correct_answer="3" label="Integer #1"/><br/>
+          <textline size="40" correct_answer="7" label="Integer #2"/>
```

Comment:
Yeah, I wasn't sure what to do about that, so I just left what was in the template before. Sub-optimal, I admit. :-\ Does anyone know how to write a better problem?